userInFormula: thinGap.stl
bagFileName: generic.bag
benchTest: random
regionRadiusSquared: 0.25
xTrigValue: 1.5
trackingController: 0
plotAccel: 0
cppFindCost: 0
useFiniteHorizon: 0
useWarmStart: 1
stlEvaluationDt: 0.1
executionDt: 0.02
truthDt: 0.005
x0: 0.5
y0: 2.5
vx0: 0
vy0: 0
controllerR: 1
controllerQ: 1
xVelLim: 0.5
yVelLim: 0.5
xAccLim: 1
yAccLim: 1
predicateSigma: 0
xe0: 3
ye0: 2.5
xBoundLow: 0
xBoundHigh: 5
yBoundLow: 0
yBoundHigh: 5
longHorzDt: 2.1
boundLossWeight: 100000000
globalViaPoints: 4
globalPopSize: 25
globalSigmaInit: 10
globalSigmaWarmStart: 2
globalMaxIter: 20
globalTmax: 20
globalVerbose: 0
shortHorzDt: 0.2
localViaPoints: 2
localPopSize: 10
localSigmaInit: 10
localSigmaWarmStart: 2
localMaxIter: 50
localTmax: 10
localVerbose: 0
robustnessOfTruth: 100000000
